include/cuco/dynamic_map.cuh

```diff
@@ -208,7 +208,7 @@ class dynamic_map {
    * @param stream Stream used for executing the kernels
    */
   template <typename InputIt,
-            typename Hash     = cuco::murmurhash3_32<key_type>,
+            typename Hash     = cuco::xxhash_32<key_type>,
```

Comment:
Probably leave this to a separate PR where an extensive performance evaluation is needed to support this change.

Reply:
Good point. I think we also should define an alias `cuco::default_hash_function` and use that instead.

Reply:
Shall we add the default hasher to the probing scheme, i.e., `cuco::linear_probing<int32_t CGSize, typename Hash1 = cuco::default_hash_function>`?

Reply:
probing method doesn't know the data structure key type

Reply:
Yeah, I figured that when I was typing the previous comment -.- Maybe as a template template parameter?

Reply:
> Maybe as a template template parameter?

Probably not, the data structure key type shouldn't have an effect on how its probing scheme works.